database/galactic-sovereign/migrations/6_create_planets.up.sql

```diff
@@ -56,17 +51,12 @@ CREATE TABLE planet_resource_production (
 
 CREATE INDEX planet_resource_production_planet_index ON planet_resource_production (planet);
 
-CREATE TRIGGER trigger_planet_resource_production_updated_at
-  BEFORE UPDATE OR INSERT ON planet_resource_production
-  FOR EACH ROW
-  EXECUTE FUNCTION update_updated_at();
```

Comment:
In some cases it does not make sense to have the trigger to automatically update the `updated_at` column: as we use this field to keep track of when the resource production/storage/amount was updated last, we sometimes need to process actions in the past.

In such situations, it can happen that there are multiple actions to be processed: each time we want to update the production (in this case) to the latest known value at the time and not the current timestamp.

This PR fixes a lot of the usage of this field in various tables.